docs/organizer-guide/newsletter/newsletter-process.rst

```diff
@@ -0,0 +1,91 @@
+
+The top section is an Editor's letter, which just serves as a quick greeting and introduction, and touches on any big things happening in the community during the month. This usually clocks in at 100-150 words.
+
+Next comes the stories. They should be edited for length and clarity and tone, as well as to make sure they've followed the `editorial guidelines, below <#editorial-guidelines>`_.
+
+Finally, the upcoming events sections covers any upcmoing community events or conferences, and then lists out any meetups that are scheduled for the next month+. It's important to include any meetups that happen in the first week or so of the following month, since the newsletter doesn't usually go out right on the first.
+
+Gathering the meetup dates is done (for now– we're working on a better process) by going to Meetup.com and searching for "Write the Docs", selecting 'Any Distance' next to your location, and selecting 'All Meetups' from the filters in the sidebar. Then go through the list and copy out the date, location, and title for the meetup, and link it to the specific event page.
+
+When the content for the newsletter is all in place, upload the file to a new branch on GitHub in 'www/docs/blog'. Create a pull request and share in the #newsletter channel for review.
+
+Allow 1-2 days for folks to review and leave comments. (Not *everyone* has to review it, but 2-3 sets of extra eyeballs is ideal.)
+
+Resolve all comments, and then when you're ready to send it, merge your Pull Request (which will take it live on the WTD site).
+
+
+****************************************************
+Shipping the newsletter (no days before shipping :p)
+****************************************************
+
+Once you've finalized the newsletter in GitHub, you'll use MailChimp to send it out. You'll need the a MailChimp login, and to be added to the Write the Docs organization.
+
+#. Login to MailChimp
+#. Click the big 'ol **Create Campaign** button.
+#. On the next screen, choose **Create an email**.
+#. Name the campaign 'Write the Docs Newsletter - [MONTH - YEAR]'
+#. On the 'Recipient' screen, select **Saved or pre-built segment**, and then **Newletter folks**.
+#. Click **Next**.
+#. On the next screen fill the form out as follows:
+
+  * **Email Subject**: Write the Docs Newsletter - [MONTH - YEAR]
+  * **Preview  Text**: First two sentences of the newsletter (ish – whatever makes a nice intro).
+  * **From name**: Write the Docs
+  * **From email**: newsletter@writethedocs.org
+  * **Campaign URL**: edit the end of the URL to reflect the current month and year
+  * Leave **Track Opens** and **Track Clicks** checked.
+
+#. Click **Next** button.
+#. Select the 'WTD Newsletter 2017' template.
+#. Click the **Next** button.
+#. On the next screen, hover over the body of the newsletter and click the **Edit** button.
+#. Switch to Github, and open the newsletter file in 'Preview' mode.
```

Comment:
Hmm, this might not always work if we use any Sphinx specific markup, but I guess is a reasonable default.

Reply:
so far, we don't, so I'll stick with this for now